src/content/en/updates/2019/05/chrome-75-deps-rems.md

```diff
@@ -0,0 +1,36 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 75 to help you plan.
+
+{# wf_updated_on: 2019-05-02 #}
+{# wf_published_on: 2019-05-02 #}
+{# wf_tags: deprecations,removals,chrome75 #}
+{# wf_blink_components: Blink,Security,Internals>Network>FTP,Internals>Network>SSL,Blink>Payments #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 75 to help you plan. #}
+
+{% include "web/updates/_shared/see-all-dep-rem.html" %}
+
+# Deprecations and removals in Chrome 75 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+## Remove overflow: -webkit-paged-x and overflow: -webkit-paged-y
+
+These are old webkit-specific properties that allowed developers to fragment
+content over columns inside a scrollable region. They are now removed.
+
+Practically speaking aren't used. Most developers use it accidentally, and
```

Comment:
"Most developers use it accidentally" what is "it"? And you've switched from plural to singular